Sources/SimpleSalesforceSDK/Models/URLRequestBuilder.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+public struct URLRequestConfig {
+  var url: URL
+  var params: Data?
+  var httpMethod: String
+  var bearerToken: String?
+  var contentType: String
```

Comment:
Maybe have defaults for most common cases
```suggestion
  var contentType: String = "application/json"
```